modules/face/samples/facemark_demo_lbf.cpp

```diff
@@ -63,6 +66,14 @@ int main(int argc, char** argv)
     FacemarkLBF::Params params;
     params.model_filename = model_path;
     params.cascade_face = cascade_path;
+    /* Change given parameters according to need. (if not given by user, set by default - default arguments given below).
```

Comment:
params are only needed for training a new model, they have no effect at all here